README.md

```diff
@@ -213,9 +174,8 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
[OPTIONAL] This is only optional because it wasn't mentioned by the previous reviewer, but I highly recommend that you implemented the changes. The are a few things that you can improve about your README to make it more professional. Here are the key points to consider to improve your README.

-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 

For example:
![image](https://user-images.githubusercontent.com/38088818/216551345-4cb3cf19-9d8d-47d5-95be-ac8d47de0049.png)
![image](https://user-images.githubusercontent.com/38088818/216551482-2a274f75-4946-434a-97ea-746434b0a464.png)
![image](https://user-images.githubusercontent.com/38088818/216551578-aae6a149-69c8-4a13-bb22-4639defb421c.png)

- The License suggestion should be removed. ✖️ 
![image](https://user-images.githubusercontent.com/38088818/216764735-b1f61cef-490e-4ceb-9bc2-17b2909f1484.png)